### PR TITLE
Added /league/leagues/{leagueId} endpoint

### PR DESCRIPTION
**Pull request by darrendlin**
Updated the League endpoint to include /league/leagues/{leagueId}

### CODE REVIEW

**Comment by darrendlin** [comment]
Hey @pseudonym117 could you take a look at my PR?

**Comment by darrendlin** [comment]
Forgot about the tests - they should be added now and everything should work.

File: riotwatcher/_apis/LeagueApiV3.py [hunk 1]
```diff
@@ -61,6 +61,21 @@ def by_summoner(self, region, summoner_id):
             '/lol/league/v3/leagues/by-summoner/{summonerId}'.format(summonerId=summoner_id)
         )
 
+    def leagues(self, region, league_id):
```
Comment (pseudonym117) [inline riotwatcher/_apis/LeagueApiV3.py]: I think the name would be better as "by_id", as this is more consistent with the naming in the other endpoints

**Comment by pseudonym117** [comment]
everything looks good, except for one comment i added. Basically i think the method name should be "by_id" instead of "leagues" to be more consistent with the other endpoints. 

Everything else looks good.

**Comment by darrendlin** [comment]
@pseudonym117 Should be all good now

**Comment by pseudonym117** [comment]
Seems good. Will likely try to address issue #84 then upload a new version to pypi.

**Pull request merged**